helm/prometheus-rules/templates/platform/honeybadger/alerting-rules/app.rules.yml

```diff
@@ -200,3 +200,27 @@ spec:
         severity: page
         team: honeybadger
         topic: releng
+    - alert: ClusterUpgradeStuck
+      annotations:
+        description: '{{`Cluster upgrade for {{ $labels.app }} is stuck. Version mismatch detected for {{ $labels.name }} in namespace {{ $labels.namespace }}.`}}'
+        runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/app-pending-update/
+      expr: |-
+        (
+          app_operator_app_info{catalog="cluster", app=~"cluster-.*", version_mismatch="true", status="already-exists"}
```

Comment:
Just to double check - this wont trigger on other apps starting with `cluster-` right? Because the `cluster` catalog only has the provider charts in?

Reply:
Yes 👍🏻